pkgs/by-name/ho/hol/package.nix

```
@@ -29,8 +29,8 @@ stdenv.mkDerivation {
   name = "${pname}-${version}";
 
   src = fetchurl {
-    url = "mirror://sourceforge/hol/hol/${longVersion}/${holsubdir}.tar.gz";
-    sha256 = "6Mc/qsEjzxGqzt6yP6x/1Tmqpwc1UDGlwV1Gl+4pMsY=";
+    url = "https://github.com/HOL-Theorem-Prover/HOL/releases/download/trindemossen-2/hol-trindemossen-2.tar.gz";
+    sha256 = "sha256-Ciy6IaB7LqwKlZOnEw1H1IcVoSL/bfbQxoWPcZD3H3w=";
```

```suggestion
    hash = "sha256-Ciy6IaB7LqwKlZOnEw1H1IcVoSL/bfbQxoWPcZD3H3w=";
```
The `sha256` [hash attribute](https://wiki.nixos.org/wiki/Nix_Hash) in fixed-output derivations has been replaced by the new `hash` attribute, which encourages the use of [SRI hashes](https://www.w3.org/TR/SRI/).